### DISCUSSION TITLE
Noodle channel used for subtraction

### DISCUSSION THREAD

**Discussion by akandrew332** [angelolab/toffy#255] [Q&A] [answered]
I've been looking at the noodle channel and it appears to have (at least in my FOVs) contamination from the gold channel.  The noodle signal that is removed from the other channels, is it the raw noodle channel, or noodle channel compensated for gold contamination.  Put another way, could Rosetta be potentially removing gold signal twice from compensated channels, and does it matter? Below is a side by side comparison of gold and noodle signal:
![Goldinnoodlechannel](https://user-images.githubusercontent.com/19846608/189237605-f84de2bc-60f2-4138-a9b0-78ae03768e3d.JPG)

**Comment by ngreenwald** [comment, marked as answer]
Yes, the Noodle channel definitely has some gold in it, so if you make very large changes to the Noodle coefficient, you'd need to update gold as well. However, for small changes to Noodle that are part of the notebook, this likely isn't a serious issue.

**Reply by akandrew332** [reply]
Is there an easy way to remove the gold from the noodle signal before removing the noodle from the other channels?  As a side note, which channels typically see the most noodle in them?

**Reply by akandrew332** [reply]
I've looked at the matrix for which channels typically see the most noodle, so I'm good on that front.

**Reply by ngreenwald** [reply]
We thought about what you suggested, but we found it easier to set the gold coefficient to subtract whatever the noodle hadn't already

**Reply by akandrew332** [reply]
Okay, that's good to know.  So the gold coefficient removes whatever gold contamination is not removed by the noodle coefficient.  Thanks, that helps!
